includes/definitions/procurve-wa.yaml

```diff
@@ -0,0 +1,11 @@
+os: procurve-wa
+text: 'HP ProCurve Access Point'
```

Comment:
I a bit confused by the name, this doesn't appear to be an access point at all.

ftp://ftp.hp.com/pub/networking/software/ProCurve-SR-7102dl-Quick-Start-Guide.pdf

Reply:
the original post was for two devices but I don't he supplied info for both.....
"Procurve Router 7102dl and ProCurve Access Point 530" 
https://github.com/librenms/librenms/issues/7218

Reply:
Good point. He's supplied data for a router and I looked at the AP side of the title. I'll tell the OP to create two separate requests with proper data.

Reply:
It looks like this PR was mostly correct for the router.